actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -685,6 +686,15 @@ private void getUserProfile(Request actorMessage) {
       result.remove(JsonKey.MISSING_FIELDS);
       result.remove(JsonKey.COMPLETENESS);
     }
+    /*
```

Comment:
As discussed today read will be from ES. So, read API call not needed in getUserProfile.

Reply:
Read API Call is removed from getUserProfile method